ui/components/app/wallet-overview/coin-overview.tsx

```diff
@@ -455,7 +454,6 @@ export const CoinOverview = ({
             defaultSwapsToken,
             ///: END:ONLY_INCLUDE_IF
             classPrefix,
-            iconButtonClassName: `${classPrefix}-overview__icon-button`,
```

Comment:
`iconButtonClassName` we've removed all instances and the styles attached to the classnames that used this prop so it is no longer needed. We can remove it.